### ISSUE TITLE
Error: Failed to launch chrome!: symbols not found

### ISSUE THREAD

**Issue by cyberic99** [astefanutti/decktape#180]
```
$ docker run --rm astefanutti/decktape:latest reveal http://localhost output.pdf
```
```
Error: Failed to launch chrome!
Error relocating /usr/lib/chromium/chrome: FT_Get_Color_Glyph_Layer: symbol not found
Error relocating /usr/lib/chromium/chrome: FT_Palette_Select: symbol not found


TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

    at onClose (/decktape/node_modules/puppeteer/lib/Launcher.js:360:14)
    at Interface.helper.addEventListener (/decktape/node_modules/puppeteer/lib/Launcher.js:349:50)
    at Interface.emit (events.js:198:15)
    at Interface.close (readline.js:394:8)
    at Socket.onend (readline.js:172:10)
    at Socket.emit (events.js:198:15)
    at endReadableNT (_stream_readable.js:1139:12)
    at processTicksAndRejections (internal/process/task_queues.js:81:17)
```
Host OS: Fedora 29

**Comment by astefanutti** [comment]
Thanks a lot for the report. It's been fixed in e45622ea7919a8e9b4f873590f8cea8816b1472c and should be available in the next `astefanutti/decktape:latest` build.

**Comment by cyberic99** [comment]
Thanks!
